test/NLog.Extensions.Logging.Tests/NLogLoggingConfigurationTests.cs

```diff
@@ -29,6 +30,42 @@ public void LoadSimpleConfig()
             Assert.Single(logConfig.AllTargets.Where(t => t is ConsoleTarget));
             Assert.Equal("hello.txt", (logConfig.FindTargetByName("file") as FileTarget)?.FileName.Render(LogEventInfo.CreateNullEvent()));
         }
+        
+        [Fact]
+        public void LoadConfigShouldIgnoreEmptySections()
+        {
+            var appSettings = @"
+{
+  ""NLog"": {
+    ""throwConfigExceptions"": true,
+    ""variables"": {},
+    ""extensions"": [],
+    ""targets"": {},
+    ""rules"": [],
+    ""keyThatPretendsToBeAComplexStructure"": {}
```

Comment:
Nice of you to document that the change will cause spelling mistakes, or other unrecognized sections like `keyThatPretendsToBeAComplexStructure` will go unvalidated (As they are completely skipped).

Reply:
With your suggestion below this test won't pass so I removed this line and renamed the test (now it only accepts well-known keys).
Added extra test that checks that code blows up for unknown key that looks like a complex structure. 
@snakefoot 

P.S. Sorry for the delay

Reply:
Thank you for the unit-tests.